File: solutions/go/pangram/1/pangram.go
package pangram

import (
    "unicode"
)

func IsPangram(input string) bool {
	seen := make(map[rune]bool)
    for _, r := range input {
        if unicode.IsLetter(r) {
            lower := unicode.ToLower(r)
            seen[lower] = true
        }
    }
    return len(seen) == 26
}

File: solutions/csharp/pangram/1/Pangram.cs
public static class Pangram
{
    public static bool IsPangram(string input)
    {
        var seen = new HashSet<char>();
        foreach (char c in input)
        {
            if (!char.IsLetter(c))
                continue;
            seen.Add(char.ToLower(c));
        }
        return seen.Count == 26;
    }
}
